### ISSUE TITLE
Calling a language specific sheet from a non-language sheet does not work

### ISSUE THREAD

**Issue by Critical-Impact** [NotAdam/Lumina#93]
Just adding these issues so people are aware and we can track them as they'll need to be fixed before the next major dalamud release.

Easiest way to replicate:
```
GameData.GetExcelSheet<GatheringPointBase>()!.GetRow(30);
var name = gpb.GatheringType.ValueNullable?.Name ?? "test";
```

It appears to be using the GatheringPointBase's page's language which is None but GatheringType requires a language to be passed in.

At a guess we'd need to grab the module's default language but I think we should also be able to request a row of a specific language

**Comment by Haselnussbomber** [comment]
I ran into this today and reported it to Asriel in [#dalamud-dev](https://discord.com/channels/581875019861328007/860813266468732938/1300120427313500182). Apparently she already has it fixed locally and will PR it soon. 🙂

**Comment by Haselnussbomber** [comment]
This is fixed in 5.2.0 (https://github.com/NotAdam/Lumina/pull/95).

**Comment by Critical-Impact** [comment]
Awsome, closing :)
